package.json

```diff
@@ -103,6 +105,7 @@
     "fast-glob": "^3.3.2",
     "is-ci": "^3.0.1",
     "node-fetch": "^3.3.2",
+    "pnpm": "9.4.0",
```

Comment:
This was added because the repo uses a specific version of pnpm and didn’t want to change my global pnpm package

with this I can use `npx pnpm [args] ` and always use the correct version